samples/Common.c

```diff
@@ -1330,6 +1327,12 @@ STATUS signalingMessageReceived(UINT64 customData, PReceivedSignalingMessage pRe
                 pPendingMessageQueue = NULL;
             }
 
+            MUTEX_LOCK(pSampleConfiguration->streamingSessionListReadLock);
+            freeStreamingSession = TRUE;
```

Comment:
What is the purpose of this bool being between the mutex_lock? Isn't the purpose of this bool to check for goto jumps from the CHK macros?

Reply:
Addressed by moving it right after the creation